optuna/terminator/improvement/evaluator.py

```diff
@@ -59,7 +59,7 @@ def get_preprocessing(self, add_random_inputs: bool = False) -> BasePreprocessin
         ]
 
         if add_random_inputs:
-            processes += [AddRandomInputs(self._min_lcb_n_additional_trials)]
+            processes += [AddRandomInputs(self._min_lcb_n_samples)]
```

Comment:
Do you think `min_lcb_n_random_samples` (or `min_lcb_n_additional_samples`) is too long?
`min_lcb_n_samples` sounds like the total number of samples used to calculate minimum LCB.

```suggestion
            processes += [AddRandomInputs(self._min_lcb_n_random_samples)]
```

Reply:
I actually think using a long name is also fine. I rather prefer `min_lcb_n_additional_samples` to `min_lcb_n_random_samples` because a non-random sampling method may be used in the future.

Reply:
I changed the name to `min_lcb_n_additional_samples`. Please let me know if you have any other idea or concern.

Reply:
Thank you for your update. Sounds good to me.